.circleci/config.yml

```diff
@@ -53,7 +52,7 @@ jobs:
           command: docker build --progress=plain -t notary_client .
       - run:
           name: "ci"
-          command: docker run --rm -e NOTARY_BUILDTAGS --env-file buildscripts/env.list --user notary notary_client bash -c "make ci && codecov"
+          command: docker run --rm -e NOTARY_BUILDTAGS --env-file buildscripts/env.list --user notary notary_client bash -c "make static && make ci && codecov"
```

Comment:
why do we want to build the static version for ci? If you want to test static build maybe add another target

Reply:
Yes it is just to test the build, I can do it in a separate step :+1: